src/Sarif/Json.Schema.ToDotNet.targets

```diff
@@ -1,47 +0,0 @@
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
This was an old version of the file whose name ended up being `Microsoft.Json.Schema.ToDotNet.targets`. It was never used and got committed accidentally.